src/Rendering/BinkVideo.cpp

```diff
@@ -135,7 +135,7 @@ void BinkVideo::Close()
         av_frame_free(&frame);
         decoder->close(codecCtx);
         av_free(codecCtx);
-        delete framebuf;
+        // delete framebuf;

```

Comment:
TODO - don't forget about that (I am using that, because skipping screens too fast on windows causes memory errors)